### DISCUSSION TITLE
Hackmos in dubai, october 25-27

### DISCUSSION THREAD

**Discussion by dckc** [Agoric/agoric-sdk#10347] [General]
[Hackmos – The Official Cosmoverse Hackathon](https://cosmoverse.org/hackmos) dubai, october 25-27, 2024

cc @Jovonni @0xpatrickdev

**Comment by Jovonni** [comment]
Hackathon winners:

## Zeromile (concentrated liquidity layer):
https://dorahacks.io/buidl/17981

## SupportMEow (cross chain, decentralized file storage):
https://dorahacks.io/buidl/17976

## Agoric Event Registration (cross chain event ticketing system):
https://dorahacks.io/buidl/18112
